front/app/containers/Admin/projects/edit/general/messages.ts

```diff
@@ -66,12 +66,12 @@ export default defineMessages({
   },
   areasLabel: {
     id: 'app.containers.AdminPage.ProjectEdit.areasLabel',
-    defaultMessage: 'Areas',
+    defaultMessage: 'Area filter',
   },
   areasLabelTooltip: {
     id: 'app.containers.AdminPage.ProjectEdit.areasLabelTooltip',
     defaultMessage:
-      'Indicate on what geographical areas this project applies. The areas can be set {areasLabelTooltipLink}.',
+      'Projects can be filtered on the homepage using areas. Areas can be set {areasLabelTooltipLink}.',
   },
   areasLabelTooltipLinkText: {
```

Comment:
Crowdin (and translators as such) will not get notified about these changes. In this case, you need to change the ids (and ideally object keys in this file, for consistency's sake).

Reply:
Sorry I didn't reassign you. My new attempt to win this battle is here https://github.com/CitizenLabDotCo/citizenlab/pull/2043/files#diff-089475f574ec29117eea4ef9b4197152b10015876edc4969575359ef22fc668fR71

Reply:
You also need to change it on Crowdin then, if the other branch is what you're going with. And close this PR

Reply:
> You also need to change the branch on Crowdin then

Yes, did it already yesterday.